### PR TITLE
Feature/error consistency cx 2352

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
The current UI for error messages doesn't not reflect the design

# Solution
Update UI for error screens:
* all error/warning to sit directly underneath the header
* all error/warning should have left right margin of 40px on desktop and 16px mobile
* errors/warnings within the confirmation screen should have an arrow pointing to the image

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [n/a] Have new manual tests been written down?
- [x] Have tests passed locally?
- [n/a] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-410-pr-onfido-sdk-ui-onfido.surge.sh

File: webpack.config.babel.js [hunk 1]
```diff
@@ -157,7 +157,7 @@ const basePlugins = (bundle_name) => ([
     // ref: https://en.wikipedia.org/wiki/Base32
     'BASE_32_VERSION' : 'AG',
     'PRIVACY_FEATURE_ENABLED': false,
-    'LIVENESS_ENABLED': false,
+    'LIVENESS_ENABLED': true,
```
Comment (elhesuu) [inline webpack.config.babel.js]: is this intentional?

Comment (stefaniacardenas) [reply webpack.config.babel.js]: Yes, just to allow @seewah to do the acceptance. I will switch it back to `false` when it's done.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by danielspagnolo** [comment]
@stefaniacardenas the error body copy needs to be 14px with line height of 20px (it's currently 16/24px). 
Otherwise it looks great.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-410-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://410-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**